Jenkinsfile

```diff
@@ -237,13 +237,65 @@ pipeline {
                         dir("${BASE_DIR}") {
                           catchError(message: 'Beta stage', buildResult: 'SUCCESS', stageResult: 'UNSTABLE') {
                             dotnetWindows(){
-                              bat 'msbuild'
+                              bat label: 'Build', script: '.ci\\windows\\msbuild.bat'
                             }
                           }
                         }
                       }
                     }
                   }
+                  stage('Test') {
+                    steps {
+                      withGithubNotify(context: 'Test Docker - Windows', tab: 'tests') {
+                        cleanDir("${WORKSPACE}/${BASE_DIR}")
+                        unstash 'source'
+                        dir("${BASE_DIR}"){
+                          dotnetWindows(){
+                            powershell label: 'Install test tools', script: '.ci\\windows\\test-tools.ps1'
+                            bat label: 'Prepare solution', script: '.ci/windows/prepare-test.bat'
+                            bat label: 'Build', script: '.ci/windows/msbuild.bat'
+                            bat label: 'Add .netCore 2.2 To Path', script: '.ci\\windows\\add-dotnet22Path.bat'
+                            bat label: 'Test & coverage', script: '.ci/windows/test.bat'
+                            powershell label: 'Convert Test Results to junit format', script: '.ci\\windows\\convert.ps1'
+                          }
+                        }
+                      }
+                    }
+                    post {
+                      always {
+                        archiveArtifacts(allowEmptyArchive: true, artifacts: "${BASE_DIR}/target/diag.log,${BASE_DIR}/target/TestResults.xml")
```

Comment:
Do we need to archive those artifacts?

Reply:
`diag.log` is the log in case of failures,  I do not remember if it is always generated

Reply:
I'm doing exactly what was done in the previous tests.  I actually need to verify that these artifacts are translating.  @gregkalapos, @v1v any chance either of you can point me at where these results should be available?

Reply:
I don't recall though... when you run the above steps locally in the windows docker container, what files are generated on the fly?